app/scripts/models/evaluation/audit.js

```diff
@@ -21,6 +23,40 @@ angular.module('wcagReporter')
         });
     });
 
+    function updateAssertionResult (assertion, data) {
+      var testResult = data.result;
+
+      function composeImportResult (result) {
+        var composed = '\n\n';
+        composed += '*Imported finding*:\n';
+        composed += 'outcome: ' + $filter('rdfToLabel')(result.outcome);
+        if (result.description.length > 0) {
+          composed += '\n' + result.description;
+        }
+
+        console.log(composed);
+
+        return composed;
+      }
+
+      assertion.result.description += composeImportResult(testResult);
+
+      // Remove empty lines at start of description
+      assertion.result.description = assertion.result.description.replace(/^\s+/, '');
+
+      // Decide what outcome should be set.
+      // Set Failed if imported result is Failed
+      // This forces the evaluator to check the import and this is the only outcome
+      // that can be set with certainty by automatic assertors.
+      if (
```

Comment:
- [ ] This part should be moved to importController and call criterion.updateOutcome

Reply:
Later